src/main/java/abr/user_playlist_abr/UserPlaylistRequestModel.java

```diff
@@ -0,0 +1,22 @@
+package abr.user_playlist_abr;
+
+public class UserPlaylistRequestModel {
+    String userName;
+    String playListID;
```

Comment:
getter methods imply these should be private

Reply:
Fixed